docs/rst/user_guide/intro.rst

```diff
@@ -8,6 +8,7 @@ User guide
 
    basic_concepts
    onedim_solver_app
+   atmosphere/intro
    package_structure
    data_guide
    unit_guide_user
```

Comment:
Let's put modelling notes at the end after the content dedicated to using the software.

```suggestion
   basic_concepts
   onedim_solver_app
   package_structure
   data_guide
   unit_guide_user
   atmosphere/intro
```